### PR TITLE
fix: fixes files name  to maintain capitalized standard

### PR DESCRIPTION
**Pull request by PJEstrada**
On linux some imports fail during startup due to the OS being case sensitive when handling imports. This PR fixes the case for the file names causing this conflict.

### CODE REVIEW

File: src/components/organisms/stories/GroupForm.stories.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { ComponentMeta, ComponentStory } from '@storybook/react'
 
 import { defaultGroupFormValues, GroupForm } from '../GroupForm'
 
-// TODO: need to mock wallet store for this to render
+// TODO: need to mock Wallet store for this to render
```
Comment (haveanicedavid) [inline src/components/organisms/stories/GroupForm.stories.tsx:5]: I can’t test this (don’t have a linux system) but can’t imagine thsi caused issue, given it’s in a code comment?

**Review by haveanicedavid — APPROVED** [review]
Thanks for catching this! 

I’m a bit confused tbh - while I don’t have a linux system, I thought that because the imports were consistent with the casing of file names, it wouldn’t cause an issue. In other words, I thought we’d only run into this issue if I had a file `Wallet.store.ts` and elsewhere tried to import `wallet.store.ts`.

Could be something with Vite though, I’m not really sure. Either way, thanks again!

File: src/components/organisms/stories/GroupForm.stories.tsx (hunk 1 above)
Comment (JeancarloBarrios) [reply src/components/organisms/stories/GroupForm.stories.tsx:5]: Yeah It seems that it is what happened

Comment (haveanicedavid) [reply src/components/organisms/stories/GroupForm.stories.tsx:5]: that would be very strange - comments are removed from typescript at compilation IIRC.

Also strange is despite seeing the renamed files here on git, when I pull the branch down locally, nothing changes. I might need to have you test… just pushed a commit (after merging this) which changes all `store` files to use lowercase notation (ie `Wallet.store.ts` -> `wallet.store.ts`). 

@JeancarloBarrios can you pull down master and see if it builds for you? You’ll also need to run `yarn`, as I added some eslint + ts settings to try and avoid this happening again

**Comment by PJEstrada** [comment]
Pleasure! I'm also not sure about the exact problem but this issue was the one that shed some light: https://github.com/microsoft/TypeScript/issues/21736

**Comment by haveanicedavid** [comment]
I ended up going down a rabbit hole and think from here forward, i’ll use `kabab-case` fro all file names. Went ahead and pushed an update for all current files, so hopefully shouldn’t be an issue moving forward

**Pull request merged**